### DISCUSSION TITLE
Stack Update fails when trying to verify SES domain

### DISCUSSION THREAD

**Discussion by yalegria** [mozilla/hubs#4042] [Hubs Cloud Questions] [answered]
**Description**
Stack Update fails when trying to verify SES domain

**To Reproduce**
1. Perform a stack update
2. Grab new template URL
3. Follow stack update steps
4. Update fails and roll back

**Expected behavior**
I expect the update goes smoothly.

**Screenshots**
<img width="1313" alt="Screen Shot 2021-03-18 at 10 33 53 AM" src="https://user-images.githubusercontent.com/7467260/111643401-5348dd00-87d5-11eb-993a-a8235d621464.png">

**Hardware** 
 - N/A

**Additional context**
Trying to update my stack to the latest 1.1.1.

**Comment by brianpeiris** [comment]
@yalegria Moved this over to Discussions so that we can help troubleshoot. If we discover an underlying bug, we can open a new issue at that point.

**Comment by brianpeiris** [comment]
@yalegria Are you updating from 1.0.0 or 1.1.0?

**Reply by yalegria** [reply]
Yes, I am. First I tried from 1.0.0 to 1.1.1, it failed with same error. Then I tried from 1.0.0 to 1.1.0, and failed as well with same error.

**Comment by johnshaughnessy** [comment, marked as answer]
The error "Modifying service token is not allowed" suggests that our stack update steps are not _replacing_ the `ServiceToken` like the docs says it should be. 

I have not investigated this issue further but want to leave this here for whoever can dig in deeper.

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-cfn-customresource.html
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-update-behaviors.html#update-replacement

**Reply by johnshaughnessy** [reply]
@yalegria is your new stack keeping the same `AvailabilityZone` as before? It seems like changing `AvailabilityZone`s is one way to encounter this issue.

**Comment by pattersonbl2** [comment]
are you still having this issues ?
